packages/react/src/primitives/message/MessageContent.tsx

```
@@ -123,6 +126,9 @@
     case "source":
       return <Source {...part} />;
 
+    case "source":
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)